skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -426,6 +459,42 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowOptOut
+The AllowOptOut parameter indicates whether or not agents can opt in or opt out from taking calls from a hunt group.
+
+PARAMVALUE: $true | $false
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases: 
+Applicable: Skype for Business Online
+
+Required: False
+Position: Named
+Default value: $true
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RoutingMethod
+The RoutingMethod defines how agents will be called in a hunt group. If the routing method is set to Serial, then agents will be called one at a time. If the routing method is set to Attendant, then agents will be called in parallel.
```

Comment:
nit : description of Attendant Routing before Serial as Attendant is the default one.